src/main/java/CosLayer.mdl

```diff
@@ -22,6 +22,8 @@ type CosDocument extends CosObject {
 	property eofCompliesPDFA: Boolean;
 	% check matches of Info dictionary and XMP metadata if first present
 	property doesInfoMatchXMP: Boolean;
+	% link to all embedded file specifications
+	link EmbeddedFiles: CosFileSpecification
```

Comment:
Missing semicolon here, I'd have fixed this if it weren't for the reference further down.
```
[ERROR] ERROR:missing ';' at '% link to the document cross reference table\n' (file:/home/cfw/Dropbox/Desktop/veraPDF/dev/veraPDF-model/src/main/java/CosLayer.mdl line : 27)
```